includes/tracks/events/class-wc-products-tracking.php

```diff
@@ -35,4 +36,31 @@ public static function track_product_updated( $product_id, $post ) {
 
 		WC_Tracks::record_event( 'update_product', $properties );
 	}
+
+	/**
+	 * Send a Tracks event when a product category is created.
+	 *
+	 * @param int $category_id Category ID.
+	 */
+	public static function track_product_category_created( $category_id ) {
+		// Only track category creation from the edit product screen or the
+		// category management screen (which both occur via AJAX).
+		if (
+			! defined( 'DOING_AJAX' ) ||
+			empty( $_POST['action'] ) ||
+			(
+				'add-tag' !== $_POST['action'] &&      // Product Categories screen.
+				'add-product_cat' !== $_POST['action'] // Edit Product screen.
+			)
+		) {
+			return;
+		}
+
+		$properties = array(
+			'category_id' => $category_id,
+			'source'      => ( 'add-tag' === $_POST['action'] ) ? 'categories' : 'product',
```

Comment:
Just looking at the events in Hue and `page` is more intuitive than `source`. Or maybe `source => 'product page'`